operate/client/src/App/BatchOperations/BatchItemsCount.test.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {render, screen} from 'modules/testing-library';
+import {BatchItemsCount} from './BatchItemsCount';
+
+describe('<BatchItemsCount />', () => {
+  it('should render not started state when no operations have started', () => {
+    render(
+      <BatchItemsCount
+        operationsCompletedCount={0}
+        operationsFailedCount={0}
+        operationsTotalCount={10}
+      />,
+    );
+
+    expect(screen.getByText('0')).toBeInTheDocument();
+    expect(screen.getByLabelText('not started')).toBeInTheDocument();
+  });
+
+  it('should filter out items with 0 count', () => {
+    render(
+      <BatchItemsCount
+        operationsCompletedCount={5}
+        operationsFailedCount={0}
+        operationsTotalCount={10}
+      />,
+    );
+
+    // Should show successful and pending, but not failed
+    expect(screen.getByLabelText('5 successful')).toBeInTheDocument();
+    expect(screen.getByLabelText('5 pending')).toBeInTheDocument();
+    expect(screen.queryByLabelText(/failed/)).not.toBeInTheDocument();
+  });
+
+  it('should display correct counts for successful, failed, and pending operations', () => {
+    render(
+      <BatchItemsCount
+        operationsCompletedCount={15}
+        operationsFailedCount={3}
+        operationsTotalCount={25}
+      />,
+    );
+
+    // Should show all three statuses with correct counts
```

Comment:
Comments should explain the why and not the what. IMO these comments are unnecessary